app/core/api/api-clients.coffee

```diff
@@ -0,0 +1,23 @@
+fetchJson = require './fetch-json'
+
+module.exports = {
+
+  url: (clientID) -> "db/api-clients/#{clientID}/new-secret"
+  
+  post: (options={}) ->
+    fetchJson('db/api-clients', _.assign {}, {
```

Comment:
Let's add `/` to the beginning of these paths to be consistent with the rest of the code base, `fetchJson('/db`